docs/src/main/sphinx/connector/hive.rst

```diff
@@ -472,6 +472,9 @@ Property Name                           Description
 ``hive.hide-delta-lake-tables``         Controls whether to hide Delta Lake tables in table
                                         listings. Currently applies only when using the AWS Glue
                                         metastore. Default is ``false``.
+``hive.hide-iceberg-tables``            Controls whether to hide Iceberg tables in table
```

Comment:
do we need a toggle for that. It feels to me that we should always hide unless hive->iceberg table redirections are enabled.
@findepi opinion?

Reply:
in case that the non-hive tables should be _always_ hidden, please take into account the `hive.hide-delta-lake-tables` which would probably become obsolete.

Reply:
This property is basically not implemented in this PR (it's not implemented for Thrift, and is silently ignored for Glue).
We added hive.hide-delta-lake-tables because we didn't have redirects back then. 

Drop the property.